requirements.txt

```diff
@@ -0,0 +1,19 @@
+aniso8601==9.0.1 ; python_version >= "3.9" and python_version < "4.0"
+attrs==23.2.0 ; python_version >= "3.9" and python_version < "4.0"
+blinker==1.8.2 ; python_version >= "3.9" and python_version < "4.0"
+click==8.1.7 ; python_version >= "3.9" and python_version < "4.0"
+colorama==0.4.6 ; python_version >= "3.9" and python_version < "4.0" and platform_system == "Windows"
+flask-restx==1.3.0 ; python_version >= "3.9" and python_version < "4.0"
+flask==3.0.3 ; python_version >= "3.9" and python_version < "4.0"
+importlib-metadata==8.0.0 ; python_version >= "3.9" and python_version < "3.10"
+itsdangerous==2.2.0 ; python_version >= "3.9" and python_version < "4.0"
+jinja2==3.1.4 ; python_version >= "3.9" and python_version < "4.0"
+jsonschema-specifications==2023.12.1 ; python_version >= "3.9" and python_version < "4.0"
+jsonschema==4.23.0 ; python_version >= "3.9" and python_version < "4.0"
+markupsafe==2.1.5 ; python_version >= "3.9" and python_version < "4.0"
+pytz==2024.1 ; python_version >= "3.9" and python_version < "4.0"
+referencing==0.35.1 ; python_version >= "3.9" and python_version < "4.0"
+rpds-py==0.19.0 ; python_version >= "3.9" and python_version < "4.0"
+werkzeug==3.0.3 ; python_version >= "3.9" and python_version < "4.0"
+zipp==3.19.2 ; python_version >= "3.9" and python_version < "3.10"
+pytest==8.3.2
```

Comment:
(If on the other hand it is important, presumably we want the restrictions here too!)